README.md

```diff
@@ -132,12 +132,13 @@ for best compatibility with external dependencies:
 
 ```js
 // webpack.config.js
+const path = require('path');
 
 module.exports = {
   // ...
   output: {
     libraryTarget: 'commonjs',
-    path: '.webpack',
+    path: path.resolve(__dirname, '.webpack'),
     filename: 'handler.js', // this should match the first part of function handler in `serverless.yml`
```

Comment:
Since 3.x.x the filename should also be defined as `[name].js` so that it can be used with automatic entry resolution, which is now the preferred way to set the entries.

Reply:
Changed it right now - ready to merge 😄